lib/servo.js

```diff
@@ -97,7 +105,9 @@ function Servo(opts) {
     this, opts = Board.Options(opts)
   );
 
-  this.range = opts.range || [0, 180];
+  this.deviceRange = opts.deviceRange || [0, 180];
```

Comment:
The value is always an array of degrees? What about `degreeRange`?

Reply:
I don't have strong feelings either way. degreeRange works for me.